### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Ddataa**
## Problem

- Profile picture is not showing because of CORB error and same for uploaded files

## Solution

- problematic quotation in nginx config at `Access-Control-Allow-Headers` and probably `Access-Control-Expose-Headers`. solved some warnings from `Strict-Transport-Security` too

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Ddataa** [comment]
By the power of systemd, I invoke The Great App CI to test this Pull Request!

File: conf/nginx.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ set $allowed_origins "https://${sandbox_domain}";
 set $api_domain "__DOMAIN__";
 set $files_domain "__DOMAIN__";
 ssl_ecdh_curve secp384r1;
-more_set_headers "Strict-Transport-Security: 'max-age=31536000; includeSubDomains' always";
+more_set_headers "Strict-Transport-Security: max-age=31536000; includeSubDomains; always";
 more_set_headers "X-XSS-Protection: '1; mode=block'";
 more_set_headers "X-Content-Type-Options: nosniff";
```
Comment (alexAubin) [inline conf/nginx.conf:9]: Not super comfortable with those three lines :/ There's already https://github.com/YunoHost/yunohost/blob/dev/conf/nginx/server.tpl.conf#L52 and https://github.com/YunoHost/yunohost/blob/dev/conf/nginx/security.conf.inc#L33-L34

Comment (Ddataa) [reply conf/nginx.conf:9]: redundant you think ?

**Review by alexAubin — APPROVED** [review]
Super quick review, LGTM though I don't know much about the details :sweat_smile:

File: conf/nginx.conf (hunk 1 above)
Comment (Ddataa) [reply conf/nginx.conf:9]: it needs to be tested, can you put this in another PR ? please

**Pull request merged**